bundles/org.openhab.binding.mercedesme/src/main/resources/OH-INF/thing/charge-channel-types.xml

```diff
@@ -101,9 +101,9 @@
 		<state pattern="%1$td.%1$tm. %1$tH:%1$tM" readOnly="true"/>
 	</channel-type>
 	<channel-type id="max-soc">
-		<item-type>Number:Dimensionless</item-type>
+		<item-type unitHint="%">Number:Dimensionless</item-type>
 		<label>Charge SoC Maximum</label>
-		<state pattern="%d %%"/>
+		<state pattern="%d %unit%"/>
```

Comment:
Not sure if this is going to work. As the optionlist contains % as unit, setting the unit to anything different (like One) would cause issues.

Reply:
I had enough [beef with UOM](https://community.openhab.org/t/units-of-measure-oh-chart/149991/3?) since the beginning. 
It's not a _**hint**_ to configure the max charge limit - it's a **_fact_** to configure as % and not `one, db, ppm` or whatever the user is capable to do!
If you choose something on your own - garbage in, garbage out

Do you have any advice to do it correctly?

Reply:
I would just revert this the way it was, percentage is the only correct unit.

Reply:
Isn't `One` also, i.e. 0.67 rather than 67%, if this is preferred?

Reply:
> Isn't `One` also, i.e. 0.67 rather than 67%, if this is preferred?

Correct, it's shown this way.
![image](https://github.com/user-attachments/assets/1f34a274-d397-4b15-8683-7e8c9bc541a7)

> I would just revert this the way it was, percentage is the only correct unit.

The user can choose unit `one` and it shall not been shown as `0.8 %`
In above screenshot you can see there's no unit displayed if unit `one` is choosed.
And if users chooses unit ppm then the state pattern will show `800000 ppm`

Looks good for me, I would prefer to leave it this way.

Reply:
Oke agree it fits.
Maybe it was more or less my personall preference. When i think of a percentage, i see a 67% and not 0,67 as that is just a decimal that could be anything.